src/doc/rustc/src/platform-support.md

```diff
@@ -30,14 +30,14 @@ All tier 1 targets with host tools support the full standard library.
 
 target | notes
 -------|-------
-`aarch64-unknown-linux-gnu` | ARM64 Linux (kernel 4.2, glibc 2.17+) [^missing-stack-probes]
+`aarch64-unknown-linux-gnu` | ARM64 Linux (kernel 4.1, glibc 2.17+) [^missing-stack-probes]
```

Comment:
Note that this one actually went backwards because crosstool-ng 1.24 doesn't have kernel 4.2, but the nearest are 4.1.49 or 4.4.174. It doesn't make any practical difference AFAIK.

Reply:
Hmmm. 4.4 is Ubuntu 16 and is "Super LTS", so if we rolled forward here it would be easier to test compliance.

And, well, there were also significant improvements in the Linux kernel's arm64 support in 4.2, 4.3, and 4.4, in particular in atomics and page support. so I think it might actually be better to roll forward, if we have to move one way or another.

Reply:
Many modern Asus wifi6/ax routers are running the 4.1 kernel so I'm glad and appreciate that 4.1 is the baseline for aarch64. I hope it remains there until such time all linux targets are moved beyond this version.